src/VirtualizedListRender.js

```diff
@@ -25,15 +25,27 @@ export default {
         itemHeight: {
             type: Number,
             required: true
+        },
+        firstItemToRender: { // index of the first item to render
```

Comment:
Why are you changing `firstItemToRender` and `lastItemToRender` from data to props?
Those variables are changed in the component when scrolling so they cannot be props.

Do you wanted to add a couple of props to select the "starting rendering state"? (I mean the list of items to render at start... I taken for granted that on the first render the firstItemToRender it's gonna be the first one)